connector-servicenow/src/main/java/com/redhat/cloud/notifications/connector/servicenow/ServiceNowRouteBuilder.java

```diff
@@ -48,8 +48,8 @@ public void configureRoute() {
 
     private void configureHttpsComponent() {
         HttpComponent httpComponent = getCamelContext().getComponent("https", HttpComponent.class);
-        httpComponent.setConnectTimeout(connectorConfig.getHttpsConnectTimeout());
-        httpComponent.setSocketTimeout(connectorConfig.getHttpsSocketTimeout());
+        httpComponent.setConnectTimeout(Timeout.ofMilliseconds(connectorConfig.getHttpsConnectTimeout()));
```

Comment:
We need to build a `Timeout` object to pass a timeout value to Camel now. There's a `Timeout#of(Duration)` method we could use (later) instead of relying on milliseconds.